src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeAnaylsisHelper.cs

```diff
@@ -0,0 +1,37 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.MSBuild;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.MSIdentity.CodeReaderWriter
+{
+    public class CodeAnaylsisHelper
+    {
+        private string _projectFilePath;
+        public CodeAnaylsisHelper(string projectFilePath)
+        {
+            _projectFilePath = projectFilePath;
+        }
+
+        private CodeAnalysis.Project? _codeAnalysisProject;
+        public CodeAnalysis.Project CodeAnalysisProject
```

Comment:
nit: Since you have CodeAnalysisHelper already you don't need CodeAnalysis iin its properties or method names for simplicity, they are already inferred from teh class name...